media-library/overview/search-update-and-delete.md

```diff
@@ -67,6 +67,20 @@ There are two places where you can add a new search query -
 1. Clicking on the `+` icon next to an existing filter lets you combine other search parameters in the same filter.
 2. Clicking on `+ Add Filter` link lets you add another filter in parallel to the existing one. This new filter can be combined either to narrow (`and` operator) or to expand (`or` operator) the results from other search filters.
 
+### Visual Search
+
+
+{% hint style="info" %}
+**Note**
+
+This feature is only available on request for specific plans.\
```

Comment:
Use standard wording.  `This feature is only available in custom enterprise pricing plans.`